DQM/L1TMonitor/src/L1TStage2RegionalMuonCandComp.cc

```diff
@@ -72,8 +81,12 @@ void L1TStage2RegionalMuonCandComp::bookHistograms(DQMStore::IBooker& ibooker, c
   summary->setBinLabel(PROCBAD, "processor mismatch", 1);
   summary->setBinLabel(TFBAD, "track finder type mismatch", 1);
   summary->setBinLabel(TRACKADDRBAD, "track address mismatch", 1);
+  if (kalman){
+    summary->setBinLabel(DXYBAD, "DXY mismatch", 1);
+    summary->setBinLabel(PT2BAD, "P_{T}2 mismatch", 1);
+  }
 
-  errorSummaryNum = ibooker.book1D("errorSummaryNum", (summaryTitle+trkAddrIgnoreText).c_str(), 14, 1, 15); // range to match bin numbering
+  errorSummaryNum = ibooker.book1D("errorSummaryNum", (summaryTitle+trkAddrIgnoreText).c_str(), nbins-3, 1, nbins-2); // range to match bin numbering
```

Comment:
Better to reassign nbins with the initial value for `errorSummaryNum`. It is not guaranteed that it will remain nbins-3 from the `summary` nbins forever.
Just do this again:
```
nbins = 14;
  if (kalman) {
    nbins += 2;
  }
```